ambry-store/src/main/java/com/github/ambry/store/DiskManager.java

```diff
@@ -127,6 +129,7 @@ public class DiskManager {
     this.time = time;
     diskIOScheduler = new DiskIOScheduler(getThrottlers(storeConfig, time));
     longLivedTaskScheduler = Utils.newScheduler(1, true);
+    indexPersistScheduler = Utils.newScheduler(1, "index-persistor-for-disk-" + disk.getMountPath(), false);
```

Comment:
you have to shutdown this scheduler in the shutdown method.

Reply:
Thank you for catching it. Updated the PR.